src/nidcpower/custom_types/lcr_measurement.py

```diff
@@ -142,11 +142,17 @@ def __init__(self, data):
         self.vdc = data.vdc
         self.idc = data.idc
         self.stimulus_frequency = data.stimulus_frequency
-        self.ac_voltage = complex(data.ac_voltage_real, data.ac_voltage_imaginary)
-        self.ac_current = complex(data.ac_current_real, data.ac_current_imaginary)
-        self.z = complex(data.z_real, data.z_imaginary)
+        if hasattr(data, "ac_voltage_real"):
```

Comment:
The `if` and `else` clauses look identical to me. What's going on here?

Reply:
One is `ac_voltage_real`; the other is `ac_voltage.real` (with a `.` before `real` instead of a `_`).

In the proto, `ac_voltage` is a `message` with `real` and `imaginary` members.